content/applications/essentials/activities.rst

```diff
@@ -2,20 +2,23 @@
 Activities
 ==========
 
+.. |clock| replace:: :icon:`fa-clock-o` :guilabel:`(clock)` icon
+
 *Activities* are follow-up tasks tied to a record in an Odoo database.
 
 .. _activities/important:
 
 The icon used to display activities varies depending on the :ref:`activity type <activities/types>`:
 
-- :guilabel:`🕘 (clock)`: the default icon, which appears if nothing is scheduled.
-- :guilabel:`📞 (phone)`: this icon appears if a phone call is scheduled.
-- :guilabel:`✉️ (envelope)`: this icon appears if an email is scheduled.
-- :guilabel:`✔️ (checkmark)`: this icon appears if a to-do is scheduled.
-- :guilabel:`(group of people)`: this icon appears if a meeting is scheduled.
-- :guilabel:`(arrow pointing up above a line)`: this icon appears if a document is scheduled to be
+- :icon:`fa-clock-o` :guilabel:`(clock)` icon: the default activities icon.
+- :icon:`fa-phone` :guilabel:`(phone)` icon: this icon appears if a phone call is scheduled.
+- :icon:`fa-envelope` :guilabel:`(envelope)` icon: this icon appears if an email is scheduled.
+- :icon:`fa-check` :guilabel:`(check)` icon: this icon appears if a to-do is scheduled.
+- :icon:`fa-users` :guilabel:`(people)` icon: this icon appears if a meeting is scheduled.
+- :icon:`fa-upload` :guilabel:`(upload)` icon: this icon appears if a document is scheduled to be
   uploaded.
```

Comment:
```suggestion
- :icon:`fa-clock-o` :guilabel:`(clock)` icon: the default activities icon.
- :icon:`fa-phone` :guilabel:`(phone)` icon: a phone call is scheduled.
- :icon:`fa-envelope` :guilabel:`(envelope)` icon: an email is scheduled.
- :icon:`fa-check` :guilabel:`(check)` icon: a "to-do" is scheduled.
- :icon:`fa-users` :guilabel:`(people)` icon: a meeting is scheduled.
- :icon:`fa-upload` :guilabel:`(upload)` icon: a document is scheduled to be
  uploaded.
```
suggestion to cut down on repetitive language, thus creating slightly more succinct definitions per icon